word_count/word_count.py

```diff
@@ -0,0 +1,34 @@
+from string import punctuation, digits
+
+
+def word_count(phrase):
+    """Count the words in a given phrase"""
+
+    phrase = validate_phrase(phrase)
+    word_count_dictionary = {}
+
+    if len(phrase) < 1:
+        return 0
+
+    for word in phrase.split(" "):
+        word_count_dictionary[word] = word_count_dictionary.get(word, 0) + 1
+        if word == "":
+            del word_count_dictionary[word]
+    return word_count_dictionary
+
+
+def validate_phrase(phrase):
+    """Remove punctuation and digits; lowercase all words"""
+
+    updated_phrase = ""
+    for words in phrase.lower():
+        if words in punctuation:
+            if words == "-":
+                words.replace("-", "")
+            updated_phrase += words.strip(punctuation)
+        elif words in digits:
```

Comment:
@paulghaddad Here I check if there are numbers in the phrase. If there are, I want to skip them. However, when I use 'continue', I get a space. I've tried removing this space by using .strip(" "), but the space remains. So I delete it from the dictionary in line 15. 

Is this the best way to go about it?

Reply:
Probably not...are you sure this is where the space is introduced? Have you used `pdb`?